### PR TITLE
[29718] Fix costlog link

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.com/wp/29718

### CODE REVIEW

File: modules/costs/app/controllers/costlog_controller.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def create
     elsif @cost_entry.save
 
       flash[:notice] = t(:notice_cost_logged_successfully)
-      redirect_back fallback_location: work_package_path(@cost_entry.work_package)
+      redirect_to work_package_path(@cost_entry.work_package)
```
Comment (ulferts) [inline modules/costs/app/controllers/costlog_controller.rb:50]: Why that change? It does not look to be related.

Comment (oliverguenther) [reply modules/costs/app/controllers/costlog_controller.rb:50]: Because redirecting back results in `GET /projects/X/cost_entries/` which do not exist
